Lib/ast.py

```diff
@@ -636,11 +660,10 @@ def visit_Expr(self, node):
         self.traverse(node.value)
 
     def visit_NamedExpr(self, node):
-        self.write("(")
-        self.traverse(node.target)
-        self.write(" := ")
-        self.traverse(node.value)
-        self.write(")")
+        with self.delimit():
```

Comment:
Maybe it would be more readable if "()" is always passed explicitly. @pablogsal: what do you think?

Reply:
I agree, I don't think delimiter should not have a default.